assets/src/edit-story/components/panels/layer/layerList.js

```diff
@@ -65,6 +79,8 @@ function LayerPanel({ layers }) {
         })
       }
     >
+      <LayerScroller direction={-1} />
```

Comment:
I don't think we need to expose this to the user here, let's put it inside `<Reorderable />`

Reply:
Fixed in b52fb791.